recipe/meta.yaml

```diff
@@ -53,12 +53,7 @@ test:
   imports:
     - jupyter_client
   commands:
-    # To fix tests with an error "zmq.error.ZMQError: Too many open files" on osx64 and arm64
-    - ulimit -n 8192
     - python -m pip check
```

Comment:
If we're planning on moving everything else into `run_test.*`, we should go ahead and put `python -m pip check` in there as well. The interaction between `run_test.*` and `commands` can get weird/buggy.